spack/r/relion/package.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Relion(CMakePackage, CudaPackage):
+    """RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a
+    stand-alone computer program that employs an empirical Bayesian approach to
+    refinement of (multiple) 3D reconstructions or 2D class averages in
+    electron cryo-microscopy (cryo-EM)."""
+
+    homepage = "http://http://www2.mrc-lmb.cam.ac.uk/relion"
+    git      = "https://github.com/3dem/relion.git"
+    url      = "https://github.com/3dem/relion/archive/3.1.0.zip"
+
+    version('3.1.1', sha256='63e9b77e1ba9ec239375020ad6ff631424d1a5803cba5c608c09fd44d20b1618')
+    version('3.1.0', sha256='8a7e751fa6ebcdf9f36046499b3d88e170c4da86d5ff9ad1914b5f3d178867a8')
+
+    # 3.0.8 latest release in 3.0 branch
+    version('3.0.8', sha256='18cdd58e3a612d32413eb37e473fe8fbf06262d2ed72e42da20356f459260973')
+    version('3.0.7', sha256='a6d37248fc4d0bfc18f4badb7986dc1b6d6849baa2128b0b4dade13cb6991a99')
+
+    # relion master contains development code
+    # contains 3.0 branch code
+    version('master')
+
+    variant('gui', default=True, description="build the gui")
```

Comment:
@alecbcs I think we would need to disable cuda here, it probably needs to find this during build to work?

Reply:
Oops one line below here :)

Reply:
You're right! Let's wait to include this once we've got support for multi-package containers.